config/Config-images.in

```diff
@@ -16,6 +16,7 @@ menu "Target Images"
 			prompt "Compression"
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_apm821xx
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_ar71xx
+			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_ath79
```

Comment:
BTW, is there anything speaking against using TARGET_ath79_mikrotik here?

This will reduce the number of devices and the possible side-effects dramatically.

Reply:
I'm wondering if we could/should completely disable the initramfs generation for this inital PR?

The 4.19 initramfs with luci (i.e. the builderbot config) doesn't tftp boot, just like 5.4

Reply:
I can certainly change the above to use `TARGET_ath79_mikrotik` for now if that helps

Reply:
The advantage of TARGET_ath79_mikrotik (if it is available in that file) won't be with respect to "your" device, but due to the fact that most other ath79 devices are unaffected. So, we just don't have to care anymore what damage we cause elsewhere.

> The 4.19 initramfs with luci (i.e. the builderbot config) doesn't tftp boot, just like 5.4

luci is only added for stable releases, the snapshots are built without. However, initramfs generation can only en/disabled per-target IIRC.
One could however add the device to master and just ignore that the initramfs won't work after switch to 5.4, as it will still build, so nothing will break for the buildbots. Nevertheless, this will obviously only defer the necessity to fix it.

Reply:
> I'm wondering if we could/should completely disable the initramfs generation for this inital PR?

Initramfs generation is entirely optional if it isn't required to flash the device. If we can't make it work, let's drop it.

Reply:
I thought it is required for initial flashing, unless you sysupgrade from ar71xx ...?

Reply:
It is, so one would have to use a 19.07 initramfs build to do that.

Reply:
Okay, I wasn't aware of this.

Reply:
or sysupgrade from an existing installation - that works too.

Something currently running RouterOS would need a tftp boot first time.

Reply:
Yes initramfs is an integral part of boostrapping openwrt on a fresh (supported) RB device: it really isn't optional. @ch6574 out of curiosity you wouldn't happen to use IRC? If you are communication would probably be easier that way.

Reply:
20 years ago I did, I can see about setting something new up if that helps here